bedrock/products/urls.py

```diff
@@ -38,12 +39,12 @@
     # VPN Resource Center
     path(
         "vpn/resource-center/",
-        views.resource_center_landing_view,
+        prefer_cms(views.resource_center_landing_view),
         name="products.vpn.resource-center.landing",
     ),
     path(
         "vpn/resource-center/<slug:slug>/",
-        views.resource_center_article_view,
+        prefer_cms(views.resource_center_article_view),
```

Comment:
Note that `prefer_cms` might not be working quite right here, since I no longer see the exist of existing Fluent translations in the language selector once published, and all locales then redirect to wagtail published `en-US` page.

Reply:
Yeah i think `prefer_cms` needs to be able to be locale-aware, and I suspect it's not right now. Will take a look as part of reviewing this PR